### ISSUE TITLE
[optional] enable Encrypt-to-Self

### ISSUE THREAD

**Issue by exlumine** [LukeSmithxyz/mutt-wizard#393]
If sending encrypted email, mutt will encrpyt the email copy in `Sent Mail` with the recepients public key. This is the more secure setting but the sender cannot access the email to read it again.

To enable mutt to encrypt the email copy in `Sent Mail` with the sender's key, the respective account config file in `~/.config/mutt/accounts/` needs the following entries (for PGP users):

```
# set pgp_self_encrypt = yes    # mutt will encrypt sent mail with your own key
# set pgp_default_key = <pgp>     # replace <pgp> with a copy of the 32 digit key id of your own/default pgp key
```

This should be an option (initially commented out) in each created account-specific config file as there could be different default keys for different accounts.

mutt knows also an option for SMIME users which is similar to this configuration.

**Comment by qorg11** [comment]
This was implemented in 12e37fc.

**Comment by exlumine** [comment]
Thank you for creating something useful from my issue. :)
